### PR TITLE
Tickets/dm 21922

### PR DESCRIPTION
**Pull request by gpdf**
Hi, @loitly, can you give this a review, primarily for build-related information?

A deployment of the built document is available at https://dmtn-136.lsst.io/v/DM-21922/index.html .

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -1,3 +1,399 @@
...
+Portal modifications will be necessary to follow that change.
+This presents a management complication and schedule risk for the LSP, because of the very limited IPAC resources available for maintenance; it will be important for there to be substantial advance notice to IPAC of when any change to this protocol is rolled out.
+
+Note that because the ``exec-portal`` capability is not sufficient on its own to allow for TAP queries (where, at least, ``read-tap`` is required), the technical possibility exists for a user with an unusual set of capabilities to be able to access the Portal Aspect but not perform any TAP (including ObsTAP) queries and therefore not be able to see any LSST data.
+This is not likely to be a situation that arises in practice.
+
+Standalone Firefly Server
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In addition to the Portal Aspect application deployed in the LSP instances in Kubernetes, at this time LSST also maintains a standalone Firefly server, for image visualization purposes, on an NCSA virtual machine at `http://lsst-demo.ncsa.illinois.edu/firefly/`__.
```
Comment (loitly) [inline index.rst]: Although http works, I think we want to point people to https.

**Review by loitly — APPROVED** [review]
The build-related information looks good.

File: index.rst (hunk 1 above)
Comment (gpdf) [reply index.rst]: Fixing.

**Comment by gpdf** [comment]
Hi, @cbanek, can you have a look at this to make sure I haven't said anything wrong about deployment?

**Comment by gpdf** [comment]
(@cbanek, your review is still welcome and I'll make any changes on a new branch.)

**Pull request merged**